src/components/container/container/container.ts

```diff
@@ -0,0 +1,58 @@
+import { CSSResultGroup, html, LitElement, PropertyValueMap, TemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { toggleDatasetEntry } from '../../core/dom';
+
+import style from './container.scss?lit&inline';
+
+/**
+ * It displays its content with the default page spacing.
+ *
+ * @slot - Use the unnamed slot to add anything to the container.
+ */
+@customElement('sbb-container')
+export class SbbContainerElement extends LitElement {
+  public static override styles: CSSResultGroup = style;
+
+  /** Whether the container is expanded. */
+  @property({ type: Boolean, reflect: true }) public expanded = false;
+
+  /** Color of the container, like transparent, white etc. */
+  @property({ reflect: true }) public color: 'transparent' | 'white' | 'milk' | 'midnight' =
+    'transparent';
+
+  protected override firstUpdated(): void {
+    this._updateStickyBar();
+  }
+
+  private _updateStickyBar(): void {
+    const stickyBar = this.querySelector('sbb-sticky-bar');
+    if (stickyBar) {
+      toggleDatasetEntry(stickyBar, 'expanded', this.expanded);
+    }
+  }
+
+  protected override willUpdate(
+    _changedProperties: PropertyValueMap<any> | Map<PropertyKey, unknown>,
+  ): void {
+    if (_changedProperties.has('expanded')) {
+      this._updateStickyBar();
+    }
+  }
+
+  protected override render(): TemplateResult {
+    return html`
+      <div class="sbb-container">
+        <slot></slot>
+      </div>
+      <slot name="sticky-bar"></slot>
```

Comment:
on slotchange, the sticky bar should be updated with expanded as well. You can at anytime insert or remove a sticky bar into the container.